frontend/public/components/catalog/catalog-page.tsx

```diff
@@ -394,5 +453,25 @@ export type CatalogProps = {
   mock: boolean;
 };
 
+export type HelmChartEntries = {
+  [name: string]: Array<HelmChart>;
+};
+
+export type HelmChart = {
+  apiVersion: string;
+  appVersion: string;
+  created: string;
+  description: string;
+  digest: string;
+  home: string;
+  icon: string;
+  keywords: Array<string>;
```

Comment:
Hmm... we should visit the `array-type` rule to see if we can enforce consistency for using `string[]` or `Array<string>`. Nothing to change here, I'm just making a note because we do both everywhere.